docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker to the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow the example of the [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) to complete the guide.
+
+## Prerequisites
+
+To register a new broker to the Service Catalog, you must have:
+* [Service Catalog](https://github.com/kubernetes-incubator/service-catalog/releases) running in version `0.1.28` or higher
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) or [helm](https://github.com/helm/helm#install) installed
+* broker's `yaml` files that specify:
+  * application which implements the [Open Service Broker API](https://www.openservicebrokerapi.org/)
+  * Kubernetes service which enables the connection between a broker and an application
+  * broker registration file in which the kind of a broker is specified
+
+> **NOTE:** In case of the exemplary UPS Broker, find the application and service files [here](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker/templates). You can find the registration file under [this](https://github.com/kubernetes-incubator/service-catalog/blob/master/contrib/examples/walkthrough/ups-broker.yaml) location.
```

Comment:
You can find the registration file under [this](https://github.com/kubernetes-incubator/service-catalog/blob/master/contrib/examples/walkthrough/ups-broker.yaml) location. -> Use [this] registration file?